### PR TITLE
#2445 Added endpoint in output while keptn auth

### LINKED ISSUE #2445: Improve CLI auth command by including endpoint in output
When authenticating the Keptn CLI to the Keptn cluster, I often use something like:

```
KEPTN_ENDPOINT=http://<ENDPOINT_OF_API_GATEWAY>/api
KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)

keptn auth --endpoint=$KEPTN_ENDPOINT --api-token=$KEPTN_API_TOKEN
```
Output:
```
Starting to authenticate
Successfully authenticated
```

It would be great if the output after this command also includes the `$KEPTN_ENDPOINT`. I know it is easy to retrieve it by `echo $KEPTN_ENDPOINT` but it is an extra step that I would like to get rid of.

The output of `keptn status` would be the right solution in my mind:
```
$ keptn status
Starting to authenticate
Successfully authenticated
CLI is authenticated against the Keptn cluster http://91.xxx.1xx.182:31090/api
```

**Definition of Done:**
- [ ] `keptn auth` shows the endpoint (like the `keptn status` command)

### PR DESCRIPTION
**Pull request by ankitjain28may**
Closes #2445

### CODE REVIEW

File: cli/cmd/auth.go [hunk 1]
```diff
@@ -101,6 +101,7 @@ More precisely, the Keptn CLI stores the endpoint and API token using *pass* in
 			}
 
 			logging.PrintLog("Successfully authenticated", logging.InfoLevel)
+			logging.PrintLog("CLI is authenticated against the Keptn cluster "+*authParams.endPoint, logging.InfoLevel)
```
Comment (agrimmer) [inline cli/cmd/auth.go:104]: Thanks for your contribution!
Can we please combine those messages
`logging.PrintLog("Successfully authenticated against the Keptn cluster "+*authParams.endPoint, logging.InfoLevel)`

**Review by agrimmer — COMMENTED** [review]
Thank you for your contribution! I have a small request.

File: cli/cmd/auth.go (hunk 1 above)
Comment (ankitjain28may) [reply cli/cmd/auth.go]: @agrimmer Thanks for reviewing, have combined the messages :)

**Comment by johannes-b** [comment]
We should file a backport to 0.7.2 since @jetzlstorfer requires it.

**Comment by christian-kreuzberger-dtx** [comment]
I will handle the backport.

**Pull request merged**